src/OpenTelemetry.Instrumentation.AspNetCore/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Removes reflection for extracting required objects from DiagnosticSource event
```

Comment:
no user facing change here, so we dont need this changelog right?

Reply:
+1